website/docs/core-concepts/stacks/hooks.mdx

```diff
@@ -20,9 +20,55 @@ configuring the `hooks` section in your stack manifest for the component that yo
 The `hooks` section schema is as follows:
 
 ```yaml
+hooks:
+  store-outputs:
+    events:
+      - after-terraform-apply
+    command: store
+    name: prod/ssm
+    outputs:
+      vpc_id: .id
+```
+
+This schema can be specified at the `global`, `terraform`, `component` or `overrides` sections of the stack config.
+
+```yaml
+# global
+hooks:
+  store-outputs:
+    events:
+      - after-terraform-apply
+    command: store
+    name: prod/ssm
+    outputs:
+      vpc_id: .id
+
+# terraform
+terraform:
+  hooks:
+    store-outputs:
+      events:
+        - after-terraform-apply
+      command: store
+      name: prod/ssm
+      outputs:
+        vpc_id: .id
+
+# overrides
+overrides:
+  hooks:
+    store-outputs:
+      events:
+        - after-terraform-apply
+      command: store
+      name: prod/ssm
+      outputs:
+        vpc_id: .id
```

Comment:
IMO, it's confusing that we show the same configuration just duplicated and not how it would _probably_ be used. Or showing how it can be used to achieve DRY configuration.

Reply:
I've updated it: https://pr-1112.atmos-docs.ue2.dev.plat.cloudposse.org/core-concepts/stacks/hooks